ui/src/app/pages/homepage/homepage.tsx

```diff
@@ -302,124 +371,267 @@ export const Homepage = withUserProfile()(class extends React.Component<
+                               data-test-id={'quick-tour-resource-' + i}>
+                      <img style={{maxHeight: '7rem', width: '9rem', marginRight: '0.5rem'}}
+                           src={thumbnail.src}/>
+                    </Clickable>
+                  </React.Fragment>;
+                })}
+              </div>
+            </div>
+            <div>
+              <div style={styles.footer}>
+                <div style={styles.footerInner}>
+                  <div style={styles.footerTitle}>
+                    How to Use the All of Us Researcher Workbench</div>
+                  <div style={{display: 'flex', flexDirection: 'row',
+                    justifyContent: 'space-between'}}>
+                    {footerLinks.map((col, i) => {
+                      return <React.Fragment key={i}>
+                        <div style={styles.linksBlock}>
+                          <div style={styles.footerText}>
+                            <div style={styles.footerTextTitle}>{col.title}</div>
+                            <ul style={{color: colors.secondary, marginLeft: '2%'}}>
+                              {col.links.map((link, ii) => {
+                                return <li key={ii}>
+                                  <a href='#' style={{color: colors.secondary}}>{link}</a>
+                                </li>;
+                              } )}
+                            </ul>
+                          </div>
                         </div>
-                      </div>
-                    </React.Fragment>;
-                  })}
+                      </React.Fragment>;
+                    })}
+                  </div>
                 </div>
               </div>
             </div>
             <div style={styles.bottomBanner}>
               <div style={styles.logo}>
                 <img src='/assets/images/all-of-us-logo-footer.svg'/>
               </div>
+              <div style={styles.bottomLinks}>Copyright ©2018</div>
               <div style={styles.bottomLinks}>Privacy Policy</div>
               <div style={styles.bottomLinks}>Terms of Service</div>
             </div>
           </div>
         </div>
+        {quickTour &&
+        <QuickTourReact closeFunction={() => this.setState({quickTour: false})} />}
+        {videoOpen && <Modal width={900}>
+          <div style={{display: 'flex'}}>
+            <div style={{flexGrow: 1}}></div>
+            <Clickable onClick={() => this.setState({videoOpen: false})}>
+              <ClrIcon
+                shape='times'
+                size='24'
+                style={{color: colors.accent, marginBottom: 17}}
+              />
+            </Clickable>
+          </div>
+          <video width='100%' controls autoPlay>
+            <source src={videoLink} type='video/mp4'/>
+          </video>
+        </Modal>}
+      </React.Fragment>;
+    } else { // delete this block below once enableHomepageRestyle is removed
```

Comment:
below this block is the old homepage, to be deleted when the entire homepage restyling epic is completed